### ISSUE TITLE
Will there be specific events?

### ISSUE THREAD

**Issue by jackivan88** [seoul-christian-school/hackathon-one#8]
Will the hackathon be scheduled into specific events or will it be more open for attendees to roam? Does it need to be a blend of the two options?

**Comment by brighton818** [comment]
Yes. There will be for beginners and for advanced people. It would vary depending on their skills.

**Comment by jackivan88** [comment]
What would be some good events that will have a lot of impact in a short
amount of time? Perhaps teaching people about Tor or Tails?

**Comment by atknix** [comment]
How would we know that the events are not events that are specifically pertaining to you guys but to the public.
